packages/firebase_ui_storage/example/linux/CMakeLists.txt

```diff
@@ -0,0 +1,138 @@
+# Project-level configuration.
+cmake_minimum_required(VERSION 3.10)
+project(runner LANGUAGES CXX)
+
+# The name of the executable created for the application. Change this to change
+# the on-disk name of your application.
+set(BINARY_NAME "firebase_ui_example")
+# The unique GTK application identifier for this application. See:
+# https://wiki.gnome.org/HowDoI/ChooseApplicationID
```

Comment:
Should we remove the linux folder since we don't support it?

Reply:
we do support it via `flutterfire_desktop` 😉